### ISSUE TITLE
Strong Named Assemblies For Compatibility with .NET Framework/.NET Standard 2.0

### ISSUE THREAD

**Issue by RhodeHummel** [ChilliCream/hotchocolate#3684]
**Is your feature request related to a problem? Please describe.**
It appears that the assemblies for StrawberryShake.Transport.Http and StrawberryShake.Transport.Core are not signed. This causes issues when using .NET Framework 4.6 and up (.NET Standard 2.0) assemblies that have already been signed. Any programs that reference the unsigned library will throw an error since the Nuget packages are not signed.

**Describe the solution you'd like**
Sign the assemblies using a strong key

**Describe alternatives you've considered**
Provide a signed version of the assemblies as a separate package.

**Comment by aequi42** [comment]
Hello,

I'm using the HotChocolate server components and my Applications are also strongly named. Currently I'm allways cloning the repository and sign the assemblies by myself to update to a newer version. Are there any reasons not to do this upstream? 
Should/can I give it a try and create an PR?

Greetings
Flo

**Comment by michaelstaib** [comment]
As far as I know, there is a move to package sign rather than signing assemblies. I will have to read up on this.

**Comment by michaelstaib** [comment]
As I remembered, its a leftover of .NET Framework. 

@aequi42 why do you sign the server? It actially is better nowadays to sign the packages.

https://github.com/dotnet/runtime/blob/main/docs/project/strong-name-signing.md

**Comment by aequi42** [comment]
Thank you for your feedback!
Seems like, there was some confusion on my part 😅️
I've been running my App on .NET FX 4.8 and used HotChocolate 10, where the signed assemblies where required.
To use the HC12 I upgraded to .NET 6 and thought the dependencies still need strong names.

Upon reading further into the matter I came to the same conclusion as you, that signing Assemblies isn't necessary anymore.

> For .NET Core and .NET 5+, strong-named assemblies do not provide material benefits. The runtime never validates the strong-name signature, nor does it use the strong-name for assembly binding.  
> https://docs.microsoft.com/en-us/dotnet/standard/assembly/strong-named#why-strong-name-your-assemblies

First tests event concluded that I can reference unsigned assemblies in my "signed" application as well.

I'll try to update HC in the near future and if I'm running into issues I'll let you know.
